microsoft-edge/webview2/concepts/user-data-folder.md

```diff
@@ -1,48 +1,175 @@
+* [Get started with WebView2 in Win32 apps](../get-started/win32.md)
+* [Get started with WebView2 in WinForms apps](../get-started/winforms.md)
+* [Get started with WebView2 in WinUI 2 (UWP) apps (public preview)](../get-started/winui2.md)
+* [Get started with WebView2 in WinUI 3 (Windows App SDK)](../get-started/winui.md)
+* [Get started with WebView2 in WPF apps](../get-started/wpf.md)
+-->
+
+| Platform | Default location of the user data folder|
+| --- | --- |
+| Win32 |  |
+| WinForms |  |
+| WinUI 2 (UWP) |  |
+| WinUI 3 (Windows App SDK) |  |
+| WPF |  |
+
+
+### Runtime flag
+
+There's a Runtime flag.
+
+
+### Scenarios
+
+This article documents both scenarios:
+
+*  Primarily, this article documents the new way of doing things.  This is the simple scenario and you don't need to know the old vs. new way.
+
+*  Secondarily, this article documents the old way of doing things, in case you have an existing WebView2 app and you want to change the user data folder location per the new convention.
+
+The default functionality for the location of the user data folder for WebView2 is changing to the new way, but you might need to know what the change was from the old to the new way so that you know how to change your code.   The old way is the exception case; this article focuses on the new, default case.
+
+For a new WebView2 app, you don't have to know about the old way.
+The key difference between the old and new ways: think about data cleanup during uninstall.<!--doc that.-->
+
+<!-- as maybe stated elsewhere in this article:
+Scenarios for using user data folder, in general; not changing:
+* Uninstalling the app and you want to clean up the data.
+* Deleting the user data folder; resetting the WebView2. -->
+
+Cases where the WebView2 runtime doesn't change the default location:
+
+*  If your WebView2 app already specifies the location of the user data folder, the WebView2 Runtime doesn't change the default location.
+<!-- better?
+*  If your WebView2 app already specifies the location of the user data folder, the WebView2 Runtime doesn't change the location to use the default location instead.
+-->
+
+*  If you don't update your WebView2 SDK, the WebView2 Runtime doesn't change the default location.
+<!-- better?
+If you don't update your WebView2 SDK, the WebView2 Runtime doesn't change the location to use the default location instead.
+-->
+
+*  If you run your WebView2 app before the change<!-- clarify --> and WebView2 created the user data folder for you, the WebView2 Runtime uses that user data folder location;
+   the WebView2 Runtime doesn't change the default user data folder location.
+   <!-- better?:
+   the WebView2 Runtime doesn't change the location of the user data folder to use the default location instead.
+   -->
+
+
+### APIs to retrieve the user data folder location
+
+To retrieve the user data folder location for an existing WebView2 app, use either of the following API items:
```

Comment:
The create async method isn't used to get the UDF, it is used to SET the UDF.  We didn't have any way until we added the new property to get the actual working UDF.  So this should only call out the property for retrieving the UDF.